.travis.yml

```diff
@@ -87,3 +88,4 @@ env:
     - RUN_LAMBDA=true
     - RUN_MACROREDUCE=true
     - RUN_COMMANDLINE=true
+    - RUN_PYLINT=true
```

Comment:
Ideally, it would be nice if we can run `RUN_PYLINT=true` first, and make its passing be a dependence for `RUN_LAMBDA=true`, etc. But I don't think it's feasible with Travis?